src/util/__tests__/BrowserStorage.test.ts

```diff
@@ -0,0 +1,41 @@
+import BrowserStorage from "../BrowserStorage";
+
+describe("BrowserStorage", () => {
```

Comment:
Since localStorage is a standard interface I do not see much benefit in having this wrapper object (except of the default value resolution, which can be however also done by initializing the browser storage upfront maybe ...).

Reply:
1. It simplifies test setup
2. If we decide, for example, to store user's settings on the server or by some other means, they are now behind an interface which can be reimplemented in one place.